src/python/espressomd/visualization_opengl.pyx

```diff
@@ -1490,47 +1356,47 @@ class openGLLive:
                         self.screenshot_capture_txt) * 9.0 - 15,
                     self.specs['window_size'][1] - 15, self.screenshot_capture_txt, col)
 
-    def _draw_sysinfo_dict(self, d):
+    def _draw_sysinfo_dict(self, sysinfo_dict):
         y = 0
-        for k, v in d.items():
+        for key, value_list in sysinfo_dict.items():
             # CATEGORY TITLE
-            self._draw_text(
-                10, self.specs['window_size'][1] - 10 - 15 * y, str(k) + ":", self.text_color)
+            self._draw_text(10, self.specs['window_size'][1] - 10 - 15 * y,
+                            f"{key}:", self.text_color)
             # ITEM LIST
-            for item in v:
+            for item in value_list:
                 txt = str(item)
-                # NUMBER OF LINES
-                nl = int(len(txt) * 9.0 / self.specs['window_size'][0]) + 1
-                ch_per_line = int(self.specs['window_size'][0] / 9) - 4
-                if ch_per_line < 20:
+                n_lines = int(
+                    len(txt) * 9.0 / self.specs['window_size'][0]) + 1
+                chars_per_line = int(self.specs['window_size'][0] / 9) - 4
+                if chars_per_line < 20:
                     break
                 ls = 0
-                for li in range(nl):
+                for li in range(n_lines):
                     y += 1
-                    ltxt = txt[ls:ls + ch_per_line]
+                    line_txt = txt[ls:ls + chars_per_line]
                     self._draw_text(
-                        30, self.specs['window_size'][1] - 10 - 15 * y, ltxt, self.text_color)
-                    ls += ch_per_line
+                        30, self.specs['window_size'][1] - 10 - 15 * y,
+                        line_txt, self.text_color)
+                    ls += chars_per_line
             y += 1.5
 
-    def _draw_particle_dict(self, d, maxlen):
+    def _draw_particle_dict(self, particle_dict, max_len):
         y = 0
-        for k, v in d.items():
-            txt = "{} {} {}".format(
-                k, (maxlen - len(k)) * ' ', v)
-            self._draw_text(
-                10, self.specs['window_size'][1] - 10 - 15 * y, txt, self.text_color)
+        for k, v in particle_dict.items():
+            txt = f"{k} {(max_len - len(k)) * ' '} {v}"
+            self._draw_text(10, self.specs['window_size'][1] - 10 - 15 * y,
+                            txt, self.text_color)
             y += 1
 
     # CALLED ION WINDOW POSITION/SIZE CHANGE
```

Comment:
```suggestion
    # CALLED ON WINDOW POSITION/SIZE CHANGE
```
Also, the "etc." latin abbreviation can only be followed by a single dot in https://github.com/espressomd/espresso/blob/afd52ecc2af6f4924b9f65b1ad7e0b4c633ae3cd/src/python/espressomd/visualization_opengl.pyx#L1258
```diff
-    # COLORS, MATERIALS ETC..
+    # COLORS, MATERIALS, ETC.
```